_data/learnnavswanlake.yml

```diff
@@ -13,6 +13,9 @@
     - title: Writing Your First Ballerina Program 
       url: /learn/user-guide/getting-started/writing-your-first-ballerina-program/
       active: writing-your-first-ballerina-program
+    - title: Writing a GraphQL API with Ballerina
+      url: /learn/user-guide/getting-started/writing-a-graphql-api-with-ballerina
+      active: writing-a-graphql-api-with-ballerina
```

Comment:
Is this permalink is correct @praneesha ?

Reply:
it should be `/learn/writing-a-graphql-api-with-ballerina/".